napari_svg/layer_to_xml.py

```diff
@@ -329,6 +340,19 @@ def points_to_xml(data, meta):
         )
         layer_xml.append(element)
 
+
+        if 'array' in text['string'] and text['visible']:
```

Comment:
[Looks like](https://github.com/napari/napari-svg/actions/runs/12709152252/job/35428438939?pr=52#step:8:235) you need to check that text is not None here:

```
  >           if 'array' in text['string'] and text['visible']:
  E           TypeError: 'NoneType' object is not subscriptable
```

```suggestion
        if text is not None and 'array' in text['string'] and text['visible']:
```